src/eve/EveShip.js

```diff
@@ -1,7 +1,17 @@
+/**
+ * EveShip
+ * @property {boolean} display
+ * @property {number} boosterGain
+ * @property {Array.<EveBoosterSet>} boosters
+ * @property {Array.<EveTurretSet>} turretSets
+ * @property {Array} _turretSetsLocatorInfo
+ * @inherits EveSpaceObject
+ * @constructor
+ */
 function EveShip()
 {
     this._super.constructor.call(this);
-
+    this.display = true;
```

Comment:
In https://github.com/ccpgames/ccpwgl/pull/151 you add display property to EveSpaceObject. That would propagate to EveShip.